src/main/java/com/amazon/sqs/javamessaging/SQSConnectionFactory.java

```diff
@@ -93,46 +85,27 @@ public AmazonSQS get() {
      * Creates a SQSConnectionFactory that uses the provided AmazonSQSClientBuilder for creating AmazonSQS client connections.
```

Comment:
Does this comment need to be updated reflect the SqsClientBuilder type?

Reply:
Yes, will change it in the next revision